File: API/Extensions/Experience.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Extensions
{
    public class Experience
    {
        public int Id { get; set; }
        public required string JobTitle { get; set; }
        public required string Company { get; set; }
        public required string Location { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public ICollection<string> Responsibilities { get; set; } = new List<string>();
    }
}

File: topcv/src/models/Experience.ts
export interface Experience {
    id: number;                      // Unique identifier for the experience
    jobTitle: string;                // Job title
    company: string;                 // Company name
    location: string;                // Location of the job
    startDate: string;               // Start date of the job (ISO format)
    endDate: string;                 // End date of the job (ISO format)
    responsibilities: string[];      // List of responsibilities
}